lib/koans/08_structs.ex

```diff
@@ -0,0 +1,59 @@
+defmodule Structs do
+  use Koans
+
+  defmodule Person do
+    defstruct [:name, :age ]
+  end
+
+  koan "structs are defined and named after a module" do
+    assert %Person{}
```

Comment:
I understand the intention here but where would we leave a blank here? `assert __`  feels a bit strange to me